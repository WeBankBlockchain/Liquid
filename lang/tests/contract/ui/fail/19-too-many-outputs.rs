use liquid_lang as liquid;

#[liquid::contract]
mod noop {
    use super::*;

    #[liquid(storage)]
    struct Noop {}

    #[liquid(methods)]
    impl Noop {
        pub fn new(&mut self) {}

        pub fn noop(
            &self,
        ) -> (
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
            u8,
        ) {
        }
    }
}

fn main() {}
