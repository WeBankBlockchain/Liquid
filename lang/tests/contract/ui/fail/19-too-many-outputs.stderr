error: the number of output arguments should not exceed 16
  --> $DIR/19-too-many-outputs.rs:16:11
   |
16 |           ) -> (
   |  ___________^
17 | |             u8,
18 | |             u8,
19 | |             u8,
...  |
33 | |             u8,
34 | |         ) {
   | |_________^
